internal/apiserver/swagger_check_test.go

```diff
@@ -65,5 +65,5 @@ func TestDiffSwaggerYAML(t *testing.T) {
 	expectedSwaggerHash := sha1.New()
 	expectedSwaggerHash.Write(existingSwaggerBytes)
 
-	assert.Equal(t, actualSwaggerHash.Sum(nil), expectedSwaggerHash.Sum(nil), "The swagger generated by the code did not match the swagger.yml file in git. Did you forget to run `make reference`?")
+	// assert.Equal(t, actualSwaggerHash.Sum(nil), expectedSwaggerHash.Sum(nil), "The swagger generated by the code did not match the swagger.yml file in git. Did you forget to run `make reference`?")
```

Comment:
This should not be removed

Reply:
Bah, shouldn't have been included in the commit. Been seeing issues with it on local builds so commented out but accidentally included. New commit re-instates it.